src/notation/internal/notationinteraction.cpp

```diff
@@ -4091,7 +4095,12 @@ void NotationInteraction::moveElementSelection(MoveDirection d)
                 Measure* mb = toMeasure(currElem);
                 currElem = isLeftDirection ? mb->prevElementStaff(si, currElem) : mb->nextElementStaff(si, currElem);
             }
+
+            if (!currElem) {
```

Comment:
The problem was that `currElem` could become null, and we called the method (`currElem->findAncestor(ElementType::VBOX)`) on a null pointer in while